Source/Widgets/CesiumInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -0,0 +1,701 @@
+                    if (defined(that._tileset)) {
+                        that._tileset.colorBlendMode = val;
+                    }
+                }
+            }
+        };
+
+        createKnockoutBindings(this, tilesetOptions);
+        createKnockoutBindings(this, {
+            _tileset: {
+                default: undefined,
+                subscribe: function(tileset) {
+                    that._style = undefined;
+                    that.statsText = '';
+                    that.pickStatsText = '';
+                    that._properties = {};
+
+                    if (defined(tileset)) {
+                        // that._statsLogger = that._updateStats.bind(that, false, false);
+                        // tileset.loadProgress.addEventListener(that._statsLogger);
+                        // tileset.allTilesLoaded.addEventListener(that._statsLogger);
+
+                        tileset.readyPromise.then(function(tileset) {
+                            that._properties = tileset.properties;
+                            that._tilesetLoaded.resolve(tileset);
+                        });
+
+                        // update tileset with existing settings
+                        var settings = ['colorize',
+                                        'wireframe',
+                                        'showBoundingVolumes',
+                                        'showContentBoundingVolumes',
+                                        'showRequestVolumes',
+                                        'freezeFrame'];
+                        var length = settings.length;
+                        for (var i = 0; i < length; ++i) {
+                            knockout.getObservable(that, settings[i]).valueHasMutated();
+                        }
+
+                        // update model with existing tileset settings
+                        that.maximumSSE = tileset.maximumScreenSpaceError;
+                        that.dynamicSSE = tileset.dynamicScreenSpaceError;
+                        that.dynamicSSEDensity = tileset.dynamicScreenSpaceErrorDensity;
+                        that.dynamicSSEFactor = tileset.dynamicScreenSpaceErrorFactor;
+                        that.colorBlendMode = tileset.colorBlendMode;
+
+                        that._updateStats(false, true);
+                        that._updateStats(true, true);
+                    }
+                }
+            },
+            _feature: {
+                default: undefined,
+                subscribe: (function() {
+                    var current = {
+                        feature: undefined,
+                        color: new Color()
+                    };
+                    return function(feature) {
+                        if (current.feature !== feature) {
+                            if (defined(current.feature) &&
+                                defined(current.feature._batchTable) &&
+                                !current.feature._batchTable.isDestroyed()) {
```

Comment:
when changing tilesets, the previous selected feature's batch table gets destroyed. Since it no longer exists to restore its previous color, we need to do this check.